release_notes/49-release-notes.adoc

```diff
@@ -0,0 +1,478 @@
+
+For more information, see xref:../integration/integrate-with-email.adoc#integrate-with-email[Integrating with email].
+
+//Monitoring
+
+//ROX-28326
+[id="prometheus-custom-metrics_{context}"]
+=== {product-title-short} supports detailed security metrics
+
+{product-title-short} Central now exposes detailed security metrics on its `/metrics` API endpoint, allowing users to scrape this data by using an existing Prometheus infrastructure.
+With this feature, you can leverage deep, customizable security observability for proactive alerting and trend analysis.
+
+For more information, see xref:../configuration/monitor-acs.adoc#custom-prometheus-metrics_monitor-acs[Using custom Prometheus metrics].
+
+//Performance
+
+//ROX-28634
+[id="vuln-defs-offline-improvement_{context}"]
+=== Improved resource handling for offline vulnerability bundles
+
+{product-title-short} has improved Central's handling of offline vulnerability bundles, resulting in less pressure on Central DB and Central disk, especially in larger environments.
+
+//ROX-29060
+[id="sensor-memory-usage-improvements_{context}"]
+=== Optimized Sensor memory usage to improve performance
+
+Sensor now uses significantly less memory than earlier releases such as 4.8 in clusters with large numbers of processes listening for connections. In extreme-scale environments, for example, clusters with greater than 10 million open ports, Sensor's memory footprint related to open connections, endpoints, and process tracking has been reduced by roughly 50%. This improvement helps prevent out-of-memory (OOM) kills. For moderate workloads such as a few thousand open ports, memory savings typically range between 10 to 15%, depending on workload characteristics. 
+
+This optimization was achieved by changing how the Sensor tracks and reports updates to Central. Previously, Sensor retained full details of all open connections, endpoints, and processes when they remained active and consumed memory required to store all details. Now, Sensor stores only a fingerprint, or hash, of each object, greatly reducing memory usage. The extra CPU cost arises from computing these hashes for every object reported by Collector.
+
+[id="notable-technical-changes_{context}"]
+== Notable technical changes
+
+This release contains the following changes:
+
+//ROX-24620
+[id="image-technology-change"]
+== Product image build and release technology change
+
+If you are updating from an {product-title-short} release earlier than 4.6.10, 4.7.7, or 4.8.3, you might notice changes to container image metadata, such as container labels or Software Bill of Materials (SBOM) contents and location. This change is because product images are now built and released by using different technologies. These changes do not affect product functionality, but they might impact your third-party integrations.
```

Comment:
Is the "an" in front of RHACM a typo? When faced with an acronym that is pronounceable, the majority of readers would say the word, not say each letter.

Also had question about the capitalization of the spelled-out form of SBOM from ~line 140 in this file.